### DISCUSSION TITLE
Where to start learning NebulaGraph and nGQL

### DISCUSSION THREAD

**Discussion by adriangalilea** [vesoft-inc/nebula#5837] [Q&A]
Hello,

I wanted to learn about graph db's and after some digging I decided to try NebulaGraph, I went ahead and got it running on docker. Then I've been trying to follow the docs which are hard to follow and I don't see anywhere where I can start learning from the basics.

I also tried following along on the [nGQL overview](https://docs.nebula-graph.io/3.6.0/3.ngql-guide/1.nGQL-overview/1.overview/) but most of the commands fail, I'm thinking that perhaps the demo_basketballplayer and the one used on the guide are not the same, I also tried downloading the one from the guide and I don't know how to import it (using NebulaGraph Studio)

If anyone can point me to a good resource to get started I'd appreciate, thanks.

**Comment by QingZ11** [comment]
So, have you now installed the NebulaGraph kernel with Docker? Have you installed the accompanying visualization tool, Nebula Studio, as well?

Once you've installed the kernel, you can find some preset datasets on the page called 'schema' in Nebula Studio, including 'basketball'. You can directly click the download button to download the corresponding dataset. It comes with some pre-configured indexes for you to query the data. You can also ignore the preset index information and search for the corresponding statement usage from the documentation—yes, the one you mentioned in the text.

**Reply by adriangalilea** [reply]
Hello,

Yes I managed to do that, I downloaded the demo basketball and tried following the nGQL overview, too many commands, copy-pasted fail and I'm unable to find good documentation to grasp the principles of the language to in order to fix them or understand what's wrong.

**Reply by QingZ11** [reply]
I roughly understand what you mean; currently, Nebula's documentation is centered around syntax, explaining how this syntax can be used. However, from a user's perspective, they might prefer to start with datasets to understand what syntax can be used for certain informative tasks.

If you're interested, you might want to try this article, which discusses simple uses of things like `MATCH`. To be honest, there indeed isn't an example that stands on a certain dataset and strings together relevant queries, but maybe in the future, I could write an article about it.

**Reply by adriangalilea** [reply]
Which article?

Thank you.

**Reply by QingZ11** [reply]
https://www.nebula-graph.io/posts/visualizing-contributor-relationships

I'm sorry, I forgot to attach the link to the article. Please refer to the above URL. If you have any questions, remember to continue the discussion in the comments.
